includes/class-wc-admin-reports-interval.php

```diff
@@ -26,6 +26,44 @@ class WC_Admin_Reports_Interval {
 	 */
 	public static $sql_datetime_format = 'Y-m-d H:i:s';
 
+	/**
+	 * Converts local datetime to GMT/UTC time.
+	 *
+	 * @param string $datetime_string String representation of local datetime.
+	 * @return DateTime
+	 */
+	public static function convert_local_datetime_to_gmt( $datetime_string ) {
+		$datetime = new DateTime( $datetime_string, new DateTimeZone( wc_timezone_string() ) );
+		$datetime->setTimezone( new DateTimeZone( 'GMT' ) );
+		return $datetime;
+	}
+
+	/**
+	 * Returns default 'before' parameter for the reports.
+	 *
+	 * @return DateTime
+	 */
+	public static function default_before() {
```

Comment:
Nice idea/refactor here, allowing the different data stores to be cleaned up when setting defaults 👍.